R/get_index_regions.R

```diff
@@ -57,3 +57,21 @@ get_index_regions <- function() {
   dplyr::distinct() %>%
   dplyr::arrange("equity_market", "country", "country_iso")
 }
+
+fetch_url_html <- function(url) {
+
+  # start the headless browser and capture the DOM as HTML after JavaScript runs
+  session <- chromote::ChromoteSession$new()
+  session$Network$setUserAgentOverride(userAgent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.4 Safari/605.1.15")
```

Comment:
Non-blocking: Is this the right useragent for us to use as a scraper?

Reply:
Genuinely no idea, I just followed suit with what @cjyetman had initially done.

What factors influence the choice of useragent?